vulcan/data/_message.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+from typing import AsyncIterator, List, Union
+
+from related import ChildField, IntegerField, SequenceField, StringField, immutable
+
+from .._api_helper import FilterType
+from .._endpoints import DATA_MESSAGE
+from ..model import Attachment, DateTime, Serializable
+
+
+@immutable
+class Address(Serializable):
+    """An address.
+
+    :var str ~.login_id: login id
+    :var str ~.address_name: address name
+    :var str ~.address_hash: address hash
+    :var str ~.initials: initials
+    """
+
+    login_id: int = IntegerField(key="LoginId")
+    address_name: str = StringField(key="Address")
+    address_hash: str = StringField(key="AddressHash")
+    initials: str = StringField(key="Initials")
+
+
+@immutable
+class Message(Serializable):
+    """A message.
+
+    :var int ~.id: Message id
+    :var str ~.title: Subject of the message
+    :var `~vulcan.hebe.model.DateTime` ~.sent_datetime: Date with time when the message was sent
+    :var `~vulcan.hebe.model.DateTime` ~.read_datetime: Date with time when the message was read
+    :var int ~.status: Message status
+    :var str ~.content: Message content
+    :var `~vulcan.data.Address` ~.sender: Sender of the message
+    :var list[Address] ~.receiver: Receiver of the message
```

Comment:
List should be uppercase, also this should be plural as a message may be sent to several user's at once.

Reply:
In this API, only one recipient is always returned - logged user.

Reply:
What if this is a sent message? (by the logged user to several teachers)

Reply:
Ah yes, i forgot about that